client/src/helper-functions/rpHelpers.js

```diff
@@ -1,10 +1,11 @@
 const helper = {
   compileRelatedProductsDataToProps: (relatedProducts, relatedProductsStyles, starRating) => {
-
+    // console.log(relatedProducts, relatedProductsStyles)
```

Comment:
still needed?

Reply:
Nah, not still needed. Cleaned up in my local so next PR it will be gone